docs/dev/libs/cep.md

```diff
@@ -246,63 +333,118 @@ pattern.where(event => ... /* some condition */).or(event => ... /* or condition
 </div>
 </div>
 
-Next, we can append further states to detect complex patterns.
-We can control the contiguity of two succeeding events to be accepted by the pattern.
+##### Conditions on Contiguity
```

Comment:
I would make it a higher level section and changed Conditions on Contiguity -> Contiguity.

Also I think this is the place where inner contiguity for looping states should be described.